src/app/public/modules/repeater/repeater-item.component.ts

```diff
@@ -184,6 +211,58 @@ export class SkyRepeaterItemComponent implements OnDestroy, OnInit {
     this.inlineFormClose.emit(inlineFormCloseArgs);
   }
 
+  public moveToTop(event: Event): void {
+    event.stopPropagation();
+    this.adapterService.moveItemUp(this.elementRef, true);
+  }
+
+  public handleKeyboardEvent(event: KeyboardEvent): void {
```

Comment:
Any reason this only activates with `space` and not `enter` ?

Reply:
@Blackbaud-ToddRoberts thoughts on this? I used `space` based on the design doc but we could modify things to use `enter` or both.

Reply:
I think we should also use `enter` in addition to `space` since it's a button and that's a default trigger for buttons

Reply:
Done